pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -775,6 +775,9 @@ func (s *awsSdkEC2) DescribeInstances(request *ec2.DescribeInstancesInput) ([]*e
 		response, err := s.ec2.DescribeInstances(request)
 		if err != nil {
 			recordAwsMetric("describe_instance", 0, err)
+			if reqerr, ok := err.(awserr.RequestFailure); ok {
```

Comment:
Coud we also add the requestID in the success cases for GetById (in the current debug logs)?

Reply:
As discussed this requires changes in the AWS SDK: the RequestID isn't available in response objects handed back through the API.